### PR TITLE
css-positioning solution - Noah Lett

### PR DESCRIPTION
**Pull request by NoahLett**
css-positioning solution screenshot 1 - Noah Lett
css-positioning solution screenshot 3
css-positioning solution screenshot 2 - Noah Lett

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  bottom: 10px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
```
Comment (Zhiwutian) [inline css-positioning/absolute.css:33]: don't forget position absolute here

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one tiny update!

File: css-positioning/absolute.css (hunk 1 above)
Comment (NoahLett) [reply css-positioning/absolute.css:34]: oops! just added

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**